quasar-core/src/main/java/co/paralleluniverse/strands/channels/disruptor/StrandBlockingWaitStrategy.java

```diff
@@ -57,11 +53,13 @@ public long waitFor1(long sequence, Sequence cursorSequence, Sequence dependentS
             try {
                 for (int i = 0; (availableSequence = cursorSequence.get()) < sequence; i++) {
                     barrier.checkAlert();
-                    processorNotifyCondition.await(i);
+                    processorNotifyCondition.await(i, 1, TimeUnit.NANOSECONDS);
```

Comment:
Okay I fixed it back to processorNotifyCondition.await(i);  commit and pushed the branch again